pkg/storage/export/export/export.go

```diff
@@ -868,8 +868,8 @@ func (ctrl *VMExportController) createExporterPodManifest(vmExport *exportv1.Vir
 	podManifest.Labels = map[string]string{exportServiceLabel: vmExport.Name}
 	podManifest.Annotations = map[string]string{annCertParams: scp}
 	podManifest.Spec.SecurityContext = &corev1.PodSecurityContext{
-		RunAsNonRoot:   pointer.Bool(true),
-		FSGroup:        pointer.Int64Ptr(kvm),
+		RunAsNonRoot:   ptr.To(true),
```

Comment:
Please see https://github.com/kubevirt/kubevirt/pull/11348
It already handle some of those (or all of them)
It also uses the internal implementation which is preferable